src/main/kotlin/no/nav/syfo/api/model/VurderingResponseDTO.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.syfo.api.model
+
+import no.nav.syfo.domain.*
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class VurderingResponseDTO private constructor(
+    val uuid: UUID,
+    val personident: String,
+    val createdAt: LocalDateTime,
+    val type: VurderingType,
+    val begrunnelse: String,
+    val document: List<DocumentComponent>,
+) {
+    companion object {
+        fun createFromVurdering(vurdering: ManglendeMedvirkningVurdering) = VurderingResponseDTO(
```

Comment:
Kanskje det holder med `fromVurdering` her ? 😊
```suggestion
        fun fromVurdering(vurdering: ManglendeMedvirkningVurdering) = VurderingResponseDTO(
```